src/cluster.c

```diff
@@ -3872,6 +3872,24 @@ void clusterLogCantFailover(int reason) {
     serverLog(LL_WARNING,"Currently unable to failover: %s", msg);
 }
 
+/* This function emits a log when an election is attempted but doesn't succeed because the quorum is not reached. */
+static void clusterLogNeededquorumNotReached(int cur_vote, int cur_quorum) {
+    static time_t lastlog_time = 0;
+    static int last_vote;
+    static int last_quorum;
+
+    /* Don't log if we have same vote and quorum for some time. 
+    (logging frequency is the same as in clusterLogCantFailover()) */
+    if (last_vote == cur_vote && last_quorum == cur_quorum && 
+        time(NULL)-lastlog_time < CLUSTER_CANT_FAILOVER_RELOG_PERIOD) 
+        return;
+
+    last_vote = cur_vote;
+    last_quorum = cur_quorum;
+    lastlog_time = time(NULL);
+    serverLog(LL_WARNING, "Currently unable to failover as the majority was not reached. Needed quorum: %i. Number of votes received so far: %i", cur_quorum, cur_vote);
```

Comment:
This also isn't strictly a warning, this is just information that we are not currently able to failover as we don't have sufficient votes. 
```suggestion
    serverLog(LL_NOTICE, "Currently unable to failover as the majority was not reached. Needed quorum: %i. Number of votes received so far: %i", cur_quorum, cur_vote);
```

Reply:
I can make this change in both functions now.

Reply:
I have made the change for both the functions currently.